src/index.html

```diff
@@ -7,6 +7,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   
+  <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
We should not be changing fonts, TailwindCSS provides us with one we've been using.

Reply:
Yeah, I think @angular/material added this on installation, can remove

Reply:
Thanks!